src/rollout.rs

```diff
@@ -20,13 +20,21 @@ pub trait RolloutStrategy {
 
 /// The simplest rollout strategy, a floating point number between 0 and 1 that
 /// represents the percentage of requests which should use the experimental
-/// method. The experimental results will be compared to the control results when
-/// using the default f64 implementation.
-impl RolloutStrategy for f64 {
+/// method. The experimental results will be compared to the control results.
+pub struct Ratio(f64);
```

Comment:
`Percent`/`Percentage` feels like a slightly more common term for this in my experience. And it would be x100 in that case.